03week/airplane/css/style.css

```diff
@@ -0,0 +1,137 @@
+.NavBar{
+  background-color: #183A55;
+  border-bottom: 2px solid white;
+  position: relative;
+  color: white;
+}
+
+.Logo{
+  width: auto;
+  height: auto;
+}
+
+.NavButtons{
+  float: right;
+  margin-right: 20px;
+  margin-top: 30px;
+  text-decoration: none;
+}
+
+.NavButtons a{
+  text-decoration: none;
+  color: white;
+}
+
+.NavButtons a:hover{
+  color: grey;
+}
+
+body{
+  background-image: url(../img/airline.jpg);
+  background-size: cover;
+}
```

Comment:
background repeats when the screen is taller.. take a look at this to get a perfect bg :)

https://css-tricks.com/perfect-full-page-background-image/